<launch>  
  <arg name="robot_name" default="tianracer_01"/>
  <node pkg="tianracer_gazebo" type="follow_the_gap_2.py" name="$(arg robot_name)_follow_the_gap" ns="$(arg robot_name)" output="screen"/>
</launch>
